taigaProject/backend/service/userstory_service.py

```diff
@@ -309,7 +312,53 @@ def partial_storypoint_burndown_for_sprint_process(sprint_id, auth_token):
 
 def get_sb_coupling(sprint_id, auth_token):
     result ={}
+    userstory_ref_map={}
+
+    user_stories = []
+
+    try:
+        custom_attribute_name = "DependsOn"
+        user_stories = get_userstories_by_sprint(sprint_id, auth_token)
+        if user_stories and len(user_stories)>0:
+
+            for userstory in user_stories:
+                userstory_ref_map[userstory['ref']] = userstory['id']
+           
+           
+            for userstory in user_stories:
+
+                logging.info(f"userstory id :{userstory['id']}")         
+                logging.info(f"userstory subject :{userstory['subject']}")               
+      
+                depends_on_values=[]
+                depends_on_list=[]
+
+                custom_attribute_data = get_custom_attribute_from_userstory(userstory['id'], auth_token)
+                custom_attribute_type_id = get_custom_attribute_type_id(userstory['project'], auth_token, custom_attribute_name)
+
+                logging.info(f"custom attribute data : {custom_attribute_data}")
```

Comment:
what's happening here. Why are we loggin?

Reply:
I just wanted to display some information in the terminal which would be useful to know how the service is calculating the results